README.md

```diff
@@ -1 +1,5 @@
-# Admission-Trials
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
```suggestion
You need to add descriptive README file to your project ? [Click here for template](https://github.com/microverseinc/readme-template/) to learn how to do it.
```

Reply:
think it would be better to add descriptive README file to your project ? (https://github.com/microverseinc/readme-template/) and copy the professional standards. Please change the parts with your details